src/toxcore/onion/packet/friend_req.rs

```diff
@@ -0,0 +1,89 @@
+/*! FriendReq packet
+*/
+
+use super::*;
+use std::str;
+use crate::toxcore::toxid::NoSpam;
+use crate::toxcore::friend_connection::packet::MAX_ONION_CLIENT_DATA_SIZE;
+
+/** Friend request that can be enclosed in onion data packet and sent through onion
+path.
+
+*/
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct FriendReq {
```

Comment:
I'd call it `FriendRequest`, we use full words for other packets.

Reply:
done